cmd/commands/cli/command.go

```diff
@@ -431,6 +433,30 @@ func (c *cliApp) natStatus() (err error) {
 	} else {
 		clio.Infof("NAT traversal status: %q (error: %q)\n", status.Status, status.Error)
 	}
+
+	connStatus, err := c.tequilapi.ConnectionStatus()
+	if err != nil {
+		clio.Warn(err)
+		return
+	}
+
+	if connStatus.Status != statusNotConnected {
```

Comment:
I think you can get the same constant from package `connectionstate` -> `connectionstate.NotConnected`
So you dont have to define it again.

Reply:
Updated